src/pages/index.js

```diff
@@ -54,49 +100,91 @@ function openProfilePopup() {
   editProfilePopup.open();
 }
 
-/* ---------------------------------------------------------------------------------------------- */
-/*                                          Section Class                                         */
-/* ---------------------------------------------------------------------------------------------- */
-
-const cardListSection = new Section(
-  {
-    items: initialCards,
-    renderer: ({ name, link }) => {
-      const newCard = createCard({ name, link });
-      cardListSection.addItem(newCard);
-    },
-  },
-  cardList
-);
-
-cardListSection.renderItems();
+//! Do Not Delete
+function handleProfileFormSubmit({ title, description }) {
+  userInfo.setUserInfo(title, description);
```

Comment:
`  userInfo.setUserInfo(title, description);
`
You need to update `DOM` only in blocks `then` after a successful response from the server so that when there is an error it wouldn’t change